### ISSUE TITLE
xcoords/ycoords can go undefined in preprocessData

### ISSUE THREAD

**Issue by ereming** [cortex-lab/KiloSort#42]
Lines 57 & 58 in `preprocessData()` expect the variables `xcoords` & `ycoords` to exist, but it looks like there is only one branch (`~isempty(ops.chanMap) && ischar(ops.chanMap)`) that generates them (by loading `ops.chanMap`).

I'm running into this as I'm working on https://github.com/cortex-lab/KiloSort/issues/41. Instead of re-generating `ops` from scratch, I'm loading it from `rez`.

**Comment by marius10p** [comment]
You are right. xcoords and xc are the same thing, unless you load them from the channelMap, and some channels are not connected. 

I pushed a fix for this.
